lib/ex_doc/formatter/html/templates.ex

```diff
@@ -59,7 +59,8 @@ defmodule ExDoc.Formatter.HTML.Templates do
   def synopsis(nil), do: nil
   def synopsis(""),  do: ""
   def synopsis(doc) when is_bitstring(doc) do
-    String.split(doc, ~r/\n\s*\n/)
+    doc
+    |> String.split(~r/\n\s*\n/)
     |> hd
```

Comment:
I agree to change `hd` to `hd()` to be consistent. But, just to be clear, the *pattern matching* will avoid the issue that you mention:

> but in the unlinkely (sic) event of a having hd/2 in the future, this code can break